tests/foreman/cli/test_remoteexecution.py

```diff
@@ -278,11 +278,14 @@ def test_positive_run_default_job_template_multiple_hosts_by_ip(
         assert result['success'] == '2', output_msgs
 
     @pytest.mark.tier3
+    @pytest.mark.no_containers
```

Comment:
why we are adding this, I assume this was working for containers ? is that correct?